### DISCUSSION TITLE
kevin QA

### DISCUSSION THREAD

**Discussion by owocki** [dcgtc/dgrants#303] [Architecture & Design]
all in all i think the experience is pretty clean, nice work everyone.

some notes below.

showstoppers
- none
- 
bugs
- this svg is 1920/1080. why wouldnt it validate on create grants page https://bits.owocki.com/GGupGLW8

ux challenges:
- i cant create a grant because i cant find an image that meets the rather stringent image requirements 'png or svg format, under 512 kB, with dimensions of 1920x1080" requirement'  i've tried all of these images: some that fit the requirements some that dont https://bits.owocki.com/7KuEK2vd
- say im someone whos new to gitcoin grants. i come to this tool; what is it; how do i use it?  what is this 'rounds' thing on the landing page?
- say im someone whos new to quadratic funding.  how does matching work?

enhancements

- can we get twitter cards setup on grants pages?  this is important bc ppl share their grants on socials a lot https://bits.owocki.com/YEuOnKE5
- on the post-checkout page ( https://bits.owocki.com/jku4bqG4 ) we could use some sort of "congrats for checking out" page.  typically ppl want to see the grants they jsut funded or to share on twitter.
- on create grant, when i drop an image on https://bits.owocki.com/RBuLQ7GX i expect it to upload. but it does not.  the page doesnt knw what to do with it
- whats up with the "Logo must be in png or svg format, under 512 kB, with dimensions of 1920x1080" requirement?  I have to go photoshop my logo?  lameee
- on grant create page, would be nice if these values were prefilled from my wallet https://bits.owocki.com/04ux4rkp
- would be nice to integrate web3modal here. ppl will ask us for walletconnect in the first round for sure https://bits.owocki.com/Apu9pNko
- where can i see 'my grants'? the grants i've created..
- where can i see 'my grants'? the grants i've funded..
-

**Comment by metafraction** [comment]
Hi @owocki can you double check the svg image dimensions? I downloaded the svg image here: https://bits.owocki.com/GGupGLW8 and the dimensions show up as 1922 x 1082

also, comments on ux challenges are well received - we have discussed improving the flow on the logo image side

**Comment by owocki** [comment]
it looks like its 1920/1080 to me... hmm.
<img width="144" alt="Screen Shot 2021-09-29 at 7 58 37 AM" src="https://user-images.githubusercontent.com/513929/135283403-6251f88f-36f5-4f9c-bb37-22c5ebd14437.png">

maybe im looking at the wrong field?

regardless, i think that an end user should not have to open their image in an image editor to upload their grant image.  consider feedback: we should *consider* using a javascript tool that can crop their image to the right format/size/dimensions before upload

**Comment by phutchins** [comment]
Hey @owocki, great and well thought out feedback. Thanks! 

I'm going to convert this to a discussion so that we can identify and create issues from it as needed.

I'll address these one at a time below:
**UX Challenges | Image upload** - I've experienced the same issue and agree that we need to change the requirements here. The idea is that we want to keep the UI clean by requiring high quality images that scale appropriately but don't bog down the page load times. This in reality makes it very difficult for some to create a grant and many who are creating grants will not have a graphic designer at hand to manipulate/create their images. I've started a discussion here: https://github.com/dcgtc/dgrants/discussions/301

**UX Challenges | Understanding dGrants and Quadratic Funding** - Currently we're working on the documentation which will cover this but I also have [a section in our requirements](https://docs.google.com/document/d/148dvDZ_St066iPuOe3GvTRRi90UOAbi4mOR-WTEEYM8/edit?usp=sharing) (see section 2.4.1) to have the UI/UX self explanatory and cleanly integrate this information into the UX flow. That work is not currently in scope yet but will likely be in the coming months. 

**Enhancements | Twitter Cards** - Already on [this one](https://github.com/dcgtc/dgrants/issues/212).

**Enhancements | Congrats for checking out page** - We've talked about this and I have now added it to the requirements doc for a future epic [here](https://docs.google.com/document/d/148dvDZ_St066iPuOe3GvTRRi90UOAbi4mOR-WTEEYM8/edit#) (see section 2.4.1).

**Enhancements | Image upload from URL** - This is something that we've discussed as well. I've added it to the [requirements doc](https://docs.google.com/document/d/148dvDZ_St066iPuOe3GvTRRi90UOAbi4mOR-WTEEYM8) (see section 2.2.1) in the scope of the first grant round we run.

**Enhancements | SVG / PNG format requirements** -  This one is covered above with the Image upload topic.

**Enhancements | Grant create page prefill values** - We discussed this as well and knowing that users would not always want their address as the owner or payout, we did not auto fill them. For the time being it may be beset for the user to be explicit when selecting these. We could start a discussion around it if anyone feels strongly. I imagine we will circle back to this one in a few weeks time.

**Enhancements | Expanding wallet compatibility** - Already on this one as well. You can follow along [here](https://github.com/dcgtc/dgrants/issues/288) and [here](https://github.com/dcgtc/dgrants/issues/275).

**Enhancements | Seeing grants created or funded** - We have captured viewing grants that you have created in our current designs in [figma here](https://www.figma.com/file/sUQSYyPfooqTcVdajfZm0t/dgrants?node-id=176%3A2429). I think it is clearly a good requirement to be able to see the donations that you have made so I have added that to the [requirements doc](https://docs.google.com/document/d/148dvDZ_St066iPuOe3GvTRRi90UOAbi4mOR-WTEEYM8) (see section 2.4.1).

**Comment by owocki** [comment]
thx for making this into a discussion. 

i dont see any immediate areas that need my comment QA-wise at this time
